project/Dependencies.scala

```diff
@@ -239,6 +244,7 @@ object Dependencies {
     scalaCache,
     apacheCommonsIO,
     workspaceManager,
+    jerseyCommon,
```

Comment:
the additions of `commonsCodec` and `jerseyCommon` in these dep lists ensures that we override the transitive deps.